testing/internal/e2e/tests/static/credential_store_test.go

```diff
@@ -111,6 +151,29 @@ func TestCliStaticCredentialStore(t *testing.T) {
 	t.Logf("Successfully deleted credential store")
 }
 
+func createPrivateKeyPemFile(fileName string) error {
+	key, err := rsa.GenerateKey(rand.Reader, 2048)
+	if err != nil {
+		return err
+	}
+	return savePrivateKeyToFile(fileName, key)
+}
+
+func savePrivateKeyToFile(fileName string, key *rsa.PrivateKey) error {
+	pemFile, err := os.Create(fileName)
+	if err != nil {
+		return err
+	}
+	defer pemFile.Close()
+
+	privateKey := &pem.Block{
+		Type:  "RSA PRIVATE KEY",
+		Bytes: x509.MarshalPKCS1PrivateKey(key),
+	}
+
+	return pem.Encode(pemFile, privateKey)
+}
```

Comment:
Don't want to put it to a new package for now as it's used within this file only

Reply:
I was a little confused about the organization of these methods. Does it make sense to just make it one method?